Extensions/BBTextLabel/bbruntimeobject.js

```diff
@@ -0,0 +1,339 @@
+/**
+ * An object displaying a video on screen.
+ *
+ * For the same video resource, only one video is being created in memory (
+ * as a HTMLVideoElement). This means that two objects displaying the same
+ * video will have the same state for this video (paused/playing, current time,
+ * volume, etc...).
+ *
+ * @memberof gdjs
+ * @class BBTextRuntimeObject
+ * @extends RuntimeObject
+ */
+gdjs.BBTextRuntimeObject = function(runtimeScene, objectData) {
```

Comment:
If your object is called BBTextRuntimeObject, the file should be called `bbtextruntimeobject.js`.
I know one can be tempted to just "reduce" the name of the file by removing "text" and "label" (from BBTextLabel), but it's a solution that create more issues down the road (the file is harder to find, you can't automate code generation because things are inconsistent, you start to mix things, etc...)